osdk/src/config/mod.rs

```diff
@@ -73,12 +84,85 @@ fn apply_args_before_finalize(action_scheme: &mut ActionScheme, args: &CommonArg
     }
     if let Some(ref mut qemu) = action_scheme.qemu {
         if let Some(path) = &args.qemu_exe {
-            qemu.path = Some(path.clone());
+            let qemu_path = path.canonicalize().expect("qemu.path from cli is invalid");
```

Comment:
Here it should use `error_msg!` too. Polish the message also. A suggested one: "The QEMU path provided with argument `--qemu-exe` does not match any files."